Example/GiniVision_Example.entitlements

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.icloud-container-identifiers</key>
+	<array/>
+	<key>com.apple.developer.ubiquity-kvstore-identifier</key>
```

Comment:
What is this? Keychain?

Reply:
UIDocumentPickerViewController needs iCloud to be enabled, explained [here](https://developer.apple.com/documentation/uikit/uidocumentpickerviewcontroller).